### PR TITLE
Better Bovine Tat descriptions, removed special/universal bovine tat …

### PR DESCRIPTION
**Pull request by lowercase-donkey**
…descriptions (edit: and now Create brandDescription widget, bug fix)

Bovine tats now have enough descriptions to stand on their own, and no longer need weird kludges.  In particular under the older code, a slave with a "shoulder" bovine tat would have her arms and back described with bovine tats, as well as whatever tats they actually had. Unfortunately, slaves could look like they had "two" tats on their arm this way.  I recently tried a few fixes, but thanks to the creative descriptions of @freecitiesbandit in #892, this is no longer needed.  This branch replaces #892 completely.

This commit also allows bovine tats to be applied piecemeal just like other tats and still be described accurately, so I partically reversed my commit in #883.

TL;DR Bovine tats are now as boring as tribal tats.

### CODE REVIEW

**Comment by lowercase-donkey** [comment]
Sorry, I meant to make the second commit a new branch.  Should be simple though.
